torch/ao/quantization/fx/convert.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Dict, List, Optional, Set, Tuple, Union, Type
+from typing import Any, Dict, List, Optional, Set, Tuple, Union, Type, Callable
```

Comment:
There were various mypy errors in the master branch version of this file in my local env, so this PR also fixes these errors.  Technically some of this stuff could be in a separate PR, lmk if that's needed.